packages/editor/src/client/panel/imageUploadAndEditPanel.tsx

```diff
@@ -2,21 +2,90 @@ import React, {useState} from 'react'
 
 import {ImagedEditPanel} from './imageEditPanel'
 import {ImageUploadPanel} from './imageUploadPanel'
+import exifr from 'exifr'
 
 export interface ImageUploadAndEditPanelProps {
   onClose(): void
   onUpload(): void
 }
 
+export type ImageMetaData = {
+  [key: string]: string
```

Comment:
Can't we use the meta data type from the exifr library?

Reply:
What do you mean? Its nothing that existed... Its something that i coded and has nothing to do with an existing library... Or did i get you wrong?

Reply:
@michael-scheurer as discusted over the phone :-)